src/notifications/screens/notifications.screen.js

```diff
@@ -196,15 +234,53 @@ class Notifications extends Component {
     });
   };
 
-  navigateToThread(notification) {
-    const { markAsRead, navigation } = this.props;
+  saveContentBlockHeight = e => {
+    const { height } = e.nativeEvent.layout;
 
-    markAsRead(notification.id);
-    navigation.navigate('Issue', {
-      issueURL: notification.subject.url.replace('pulls', 'issues'),
-      isPR: notification.subject.type === 'PullRequest',
-      language: this.props.language,
-    });
+    this.setState({ contentBlockHeight: height - sizes.tabBar });
+  };
+
+  keyExtractor = (item, index) => {
```

Comment:
Hey, this is a pretty small piece of code, I think that could be added inline in the <FlatList> like that: 

`keyExtractor={(item, index) => index}`